frameworks/opennmt_lua/Dockerfile

```diff
@@ -1,14 +1,60 @@
+FROM ubuntu:14.04 as opennmt_builder
```

Comment:
This should be `ubuntu:16.04` as it is the base image of `opennmt/opennmt:latest`.